declarations/comment.js

```diff
@@ -58,7 +58,21 @@ declare type CommentContextGitHub = {
   url: string
 };
 
-declare type CommentTag = {
+declare type CommentTagBase = {
```

Comment:
Yep! I always wrote `declare type` because that's what the Flow [library definition documentation](https://flow.org/en/docs/libdefs/creation/) talks about, but apparently [declare actually doesn't mean anything](https://github.com/prettier/prettier/issues/659), which is confusing - I'd love to track down whether it has any meaning anywhere, or why it was there in the first place.